libqtile/extension/window_list.py

```diff
@@ -48,7 +48,7 @@ def list_windows(self):
         for win in windows:
             if win.group:
                 item = self.item_format.format(
-                    group=win.group.name, id=id, window=win.name)
+                    group=win.group.label or win.group.name, id=id, window=win.name)
```

Comment:
Thanks @dhannes, I'm only wondering if this would be safe as simply `group=win.group.label`, since `win.group.label` is already initialized as `self.label = name if label is None else label` in `Group.__init__()`?